### PR TITLE
Docs - extension guide - mention all "dump" debug properties

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: docs/src/main/asciidoc/writing-extensions.adoc [hunk 1]
```diff
@@ -1873,27 +1873,44 @@ The only particular aspect of writing Quarkus extensions in Eclipse is that APT
...
 
-==== Saving Application Generated Classes to Disk
+==== Dump the Generated Classes to the File System
 
-The class augmentation step of Quarkus generates classes for various purposes. Sometimes you need to view these
-classes/bytecode to debug or understand an issue. Classes that are related to application augmentation are currently held in
-memory in a runtime class loader.
+During the augmentation phase Quarkus extensions generate new and modify existing classes for various purposes. 
+Sometimes you need to inspect the generated bytecode to debug or understand an issue.
+There are three system properties that allow you to dump the classes the the filesystem:
```
Comment (gsmet) [inline docs/src/main/asciidoc/writing-extensions.adoc:1880]:
```suggestion
There are three system properties that allow you to dump the classes to the filesystem:
```

File: docs/src/main/asciidoc/writing-extensions.adoc [hunk 2]
```diff
@@ -1873,27 +1873,44 @@ The only particular aspect of writing Quarkus extensions in Eclipse is that APT
...
 
-When using the dev mode (running `./mvnw clean compile quarkus:dev`) you can have these classes written out to disk for inspection, for this, you need to specify the
-`quarkus.debug.generated-classes-dir` system property, for example:
+- `quarkus.debug.generated-classes-dir` - to dump the generated classes, such as bean metadata
+- `quarkus.debug.transformed-classes-dir` - to dump the transformed classes, i.e. Panache entitites
```
Comment (gsmet) [inline docs/src/main/asciidoc/writing-extensions.adoc:1883]:
```suggestion
- `quarkus.debug.transformed-classes-dir` - to dump the transformed classes, e.g. Panache entities
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Spotted a couple of typos.

**Comment by mkouba** [comment]
@gsmet Fixed, thanks!

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**